### PR TITLE
Add server Dell PowerEdge R240

### PR DESCRIPTION
**Pull request by davidusken**
https://www.dell.com/en-us/shop/enterprise-products/r240-1ru-server-intel/spd/poweredge-r240

### CODE REVIEW

File: device-types/Dell/PowerEdge_R240.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+is_full_depth: true
+airflow: front-to-rear
+weight: 12.2
+weight_unit: kg
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: VGA
+    type: other
+power-ports:
```
Comment (danner26) [inline device-types/Dell/PowerEdge_R240.yaml:15]: Is this PSU modular? If so, this should be modelled in a `module-bays`, and then the applicable PSU should be added to the module-types folder

File: device-types/Dell/PowerEdge_R240.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+slug: dell_poweredge_r240
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+weight: 12.2
+weight_unit: kg
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: VGA
```
Comment (danner26) [inline device-types/Dell/PowerEdge_R240.yaml:13]: VGA is not a console port, please remove this

**Review by danner26 — CHANGES_REQUESTED** [review]

**Comment by davidusken** [comment]
Ok thanks, Dell EMC PowerEdge R240 comes with 2 different options for PSUs. It is changeable but requires opening the cover, removing a screw, and disconnecting the cables. Here is a screenshot of the PSU specifications:

image

Would you prefer this to be added as modules still or have 2 different model versions of the server?

**Comment by danner26** [comment]
Do the devices have different part numbers based on which PSU is installed?

**Pull request closed without merging**